src/forwarder/FileSender.cpp

```diff
@@ -34,16 +34,16 @@ FileSender::FileSender(const std::string& xfer_option) {
 
 void FileSender::send(std::vector<std::string>& from, const fs::path& to) {
     std::ostringstream files;
-    files << "{";
+ 
     for (int i = 0; i < from.size(); i++) {
         if (i == from.size()-1) {
             files << from[i];
         }
         else {
-            files << from[i] << ",";
+            files << from[i] << " ";
         }
     }
```

Comment:
Since you use **boost** library anyway, you may consider replacing this entire `for` loop with
```
files << boost::algorithm::join(from, " ");

Reply:
I'm going to leave this as is;  I have a longer term goal to eliminate having to use the boost library.